validation/src/main/scala/hmda/validation/rules/lar/validity/V230.scala

```diff
@@ -0,0 +1,16 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.{ LoanApplicationRegister }
+import hmda.validation.dsl.Result
+import hmda.validation.rules.EditCheck
+
+object V230 extends EditCheck[LoanApplicationRegister] {
+
+  val occupancyStatusTypes = List(1, 2, 3)
+
+  def apply(lar: LoanApplicationRegister): Result = {
+    lar.loan.occupancy is containedIn(occupancyStatusTypes)
+  }
+
+  def name = "V230"
```

Comment:
Our codebase was inconsistent about this one, with some `apply` and `name` methods having `override` while others didn't. It's more explicit and shows clear intent if we use the override modifier. You are right that technically it's optional if the parent abstract class has no concrete implementation.